citations-impl/impl/src/java/org/sakaiproject/citation/impl/CitationListAccessServlet.java

```diff
@@ -532,7 +536,9 @@ protected void handleViewRequest(HttpServletRequest req, HttpServletResponse res
       			out.println("\t\t\t</table>");
        		    out.println("\t\t</div>");
 
-			    out.println("\t\t<div class=\"availabilityHeader\"></div>");
+
+			    out.println("\t\t<div class=\"availabilityHeader\" style=\"color:" +
+					    ServerConfigurationService.getString("official.institution.background.colour") + "\">" + "<strong>Availability</strong>" + "</div>");
```

Comment:
Portal skinning is all different in 11 I don't thinks this will fit in. The citations works should ideally go back before we move to 11 so probably better to use a good property name in the future. 

Don't worry about tidying it up for this pull request though.